### PR TITLE
Improve hy_list_available_hySpc_packages() (part 2)

### LINKED ISSUE #328: hy_list_available_hySpc_packages() not working
... and the unit test fails. 

I think the resulting JSON is not properly processed.

### PR DESCRIPTION
**Pull request by GegznaV**
Code in this PR unifies the GH API output for different machines.

Related to #3 and closes cbeleites/hyperSpec#328.

***
@bryanhanson and @cbeleites, could you test this new version of function on your machines and inform us if it works for you.

Otherwise, I'll need the output file of:
```r
gh_api_response <- readLines(
  "https://api.github.com/orgs/r-hyperspec/repos?per_page=100"
)
readr::write_lines(gh_api_response, file = "gh_api_response.txt")
```
with more information on what kind of machine the file was generated.

### CODE REVIEW

**Review by bryanhanson — COMMENTED** [review]
Sorry, but it does not work for me on the Mac.  I'll put additional info requested over on Slack.

**Comment by GegznaV** [comment]
Unit test message on Bryan's MAC:
```r
  Running 'testthat.R' [8s/10s]
 ERROR
Running the tests in 'tests/testthat.R' failed.
Last 13 lines of output:
  • no spc files with empty spectra available so far (1)
  • On CRAN (2)
  • skip (1)
  • TODO: implement (1)
  
  ══ Failed tests ════════════════════════════════════════════════════════════════
  ── Failure (???): hy_list_available_hySpc_packages() works ─────────────────────
  length(pkgs) > 5 is not TRUE
  
  `actual`:   FALSE
  `expected`: TRUE 
  
  [ FAIL 1 | WARN 0 | SKIP 42 | PASS 950 ]
  Error: Test failures
  Execution halted
  ```

**Comment by GegznaV** [comment]
I've removed the additional whitespace in the GH API response. Should work now. 
image



Can you test again @bryanhanson?

**Comment by bryanhanson** [comment]
This works now on my machine.

**Comment by GegznaV** [comment]
@cbeleites We are waiting for your review (and approval).

**Review by bryanhanson — APPROVED** [review]
Works now on my machine.

**Comment by cbeleites** [comment]
I get other errors locally, but #6 works now.

**Pull request merged**
